lib/router/config-router-util.js

```diff
@@ -44,7 +45,13 @@ function parseApplication(settings, locNode, provideDefaults){
   if (locNode.getValues('app_init_timeout') && 
       locNode.getValues('app_init_timeout').timeout){
     appSettings.initTimeout = locNode.getValues('app_init_timeout').timeout;  
-  }  
+  }
+
+  if (locNode.getValues('preserve_logs') &&
+      locNode.getValues('preserve_logs').enabled){
+    console.log("Preserved!");
```

Comment:
Did you mean to leave this log statement? Maybe we should reiterate the warning you have in the configuration doc?